src/ResourceManager/ServiceBus/Commands.ServiceBus/help/Get-AzureRmServiceBusTopic.md

```diff
@@ -101,18 +101,13 @@ Id                                  : /subscriptions/854d368f-1828-428f-8f3c-f2a
 Type                                : Microsoft.ServiceBus/Topic
```

Comment:
Should Location be deleted? It is in the list of deprecated properties. If so, please apply it to this file and all others with SubscriptionAttributes or TopicAttributes